src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -199,34 +199,19 @@ object EasyIngest {
       request = (datastreamId, sdo.listFiles.find(_.getName == dsSpec.contentFile)) match {
         case ("EMD", Some(file)) =>
           // Note that this would change the ingested file's checksum, but it is only for the EMD datastream, which has no checksum
-          return executeAddRequestWithReplacement(request, file, PLACEHOLDER_FOR_DMO_ID, pidDictionary(sdo.getName))
+          request.content(replacePlaceHolder(file, PLACEHOLDER_FOR_DMO_ID, pidDictionary(sdo.getName)))
```

Comment:
Fedora properly replaces `<dc:identifier>$sdo-id</dc:identifier>` in the DC data-stream which is provided inline in the fo.xml, I wonder what would happen if stage-dataset would inline the EMD too. We probably wouldn't need this special replacement here.